pkgs/tools/misc/fluent-bit/default.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgs, stdenv, fetchgit, cmake, ... }:
+
+stdenv.mkDerivation rec {
+  src = fetchgit {
+    url = https://github.com/fluent/fluent-bit;
+    sha256 = "0rmdbrhhrim80d0hwbz56d5f8rypm6h62ks3xnr0b4w987w10653";
+    rev = "8cc3a1887c3fcd6dd95a4a475fb213a0e399c222";
+  };
+  name = "fluentbit";
+  version = "1.0.6";
+  buildInputs = [ cmake ];
+  builder = ./builder.sh;
+
+  meta = with stdenv.lib; {
+    description = "fluentbit";
```

Comment:
The description should be not include the name of the software but instead describe what the software is/does.

Reply:
E.g.: "Multi-platform Log Processor and Forwarder which allows you to collect data/logs from different sources, unify and send them to multiple destinations"
Maybe just that: "Multi-platform Log Processor and Forwarder" and the rest as `longDescription`

Reply:
Even the information that it supports multiple platforms is redundant. We have `platforms` field for that.

Reply:
Good point.